app/components/modal-billing.js

```diff
@@ -1,4 +1,13 @@
 import ModalComponent from 'ghost-admin/components/modal-base';
+import {inject as service} from '@ember/service';
 
 export default ModalComponent.extend({
+    billing: service(),
+
+    actions: {
+        closeModal() {
+            this._super(arguments);
```

Comment:
Not needed for actions. `_super(...arguments)` is only needed when you're extending from a base class and want to run the logic in the base class before your own logic